src/features/auth/pages/signup/SignupForm.tsx

```diff
@@ -1,29 +1,38 @@
-import { useFocusEffect } from '@react-navigation/native'
-import { StackScreenProps } from '@react-navigation/stack'
-import React, { FunctionComponent, useState, useCallback } from 'react'
+import React, { FunctionComponent, useState } from 'react'
 import { Keyboard } from 'react-native'
+import styled from 'styled-components/native'
 
 import { useSignUp } from 'features/auth/api/useSignUp'
-import { SIGNUP_NUMBER_OF_STEPS } from 'features/auth/constants'
+import { NEW_SIGNUP_NUMBER_OF_STEPS } from 'features/auth/constants'
```

Comment:
Vu qu'on remplace l'ancien parcours, il faut garder le nom d'origine sans "NEW" non ? 
```suggestion
import { SIGNUP_NUMBER_OF_STEPS } from 'features/auth/constants'
```